docs/getting-started.md

```diff
@@ -144,6 +144,20 @@ Experiment tracker
 
 By default, Matcha will hide sensitive resource properties. If you need one of these properties, then you can add the `--show-sensitive` flag to your `get` command.
 
+# Sharing resources
+
+If you're working in a team and wish to use the same resources (e.g. the same experiment tracking server) you can share the deployed resources with Matcha.
+
+To do so, add the `matcha.config.json` file to your GitHub repository (or similar), note that this does not contain any sensitive information such as passwords or server endpoints.
```

Comment:
I wonder if something like this might be an improvement:

# &#129309; Sharing Resources

You'll notice that a configuration file is create as part of the provisioning process - it's called `matcha.config.json`. This file stores the information necessary for Matcha to link up to your provisioned resources. 

To share these resources, you need to add this file to your project repository (Github or similar).

> Note: this file does not contain any sensitive information such as passwords or server endpoints.

The other users that you're sharing the resources with will then need to ensure this configuration file exists locally, in the same directory where the file was originally created.

Matcha is smart enough to use this file to find and pull the provisioned state information, which will allow multiple users to use the provisioned resources.

## Permissions

All users accessing the same set of provisioned resources **must** have the correct level of permissions on Azure - these can be found [here](https://mymatcha.ai/azure-permissions/)